atomic_dbus.py

```diff
@@ -6,20 +6,42 @@
 from gi.repository import GLib # pylint: disable=no-name-in-module
 import slip.dbus.service
 from slip.dbus import polkit
-
-from . import Atomic
-
+from Atomic import Atomic
+from Atomic.verify import Verify
+from Atomic.storage import Storage
+#from Atomic.diff import Diff
+from Atomic.diffDbus import Diff
+from Atomic.top import Top
+from Atomic.scan import Scan
 
 class atomic_dbus(slip.dbus.service.Object):
     default_polkit_auth_required = "org.atomic.readwrite"
 
     class Args():
-        def __init__(self, image):
-            self.image = image
+        def __init__(self):
+            self.image = None
             self.recurse = False
+            self.debug = False
+            self.devices = None
+            self.driver = None
+            self.graph = None
+            self.force = None
+            self.import_location = None
+            self.export_location = None
+            self.compares = []
+            self.rootfs = None
+            self.json = False
+            self.no_files = False
+            self.names_only = False
+            self.rpms = False
+            self.verbose = False
+            self.containers = None
+            self.optional = None
+            self.d = None
+            self.n = None
 
```

Comment:
Do we need all of these defined in init, or can we just define them for each interface that needs them